index.html

```diff
@@ -58,135 +78,138 @@ <h2>I’m a software developer</h3>
             </li>
         </ul>
     </main>
-    <section id="work">
-        <div class="work-title">
-            <div>
-                <h2>My Recent Works</h2>
-            </div>
-            <div>
-                <hr>
-            </div>
-        </div>
-        <ul id="work-content">
-            <li class="project pr-1">
-                <div class="image">
-                    <img src="images/Img Placeholder.png" alt="img">
+    <section id="work-section">
```

Comment:
- [ ] please note that the work section should have a white background instead of `#e5e5e5` kindly fix it to match figma design

Reply:
Thanks!  Changes have been made